v2/internal/testcommon/kube_test_context_envtest.go

```diff
@@ -37,7 +37,7 @@ func createSharedEnvTest(cfg testConfig, namespaceResources *namespaceResources)
 	environment := envtest.Environment{
 		ErrorIfCRDPathMissing: true,
 		CRDDirectoryPaths: []string{
-			"../../config/crd/bases",
+			"../../config/crd/packed",
```

Comment:
I'm just curious -- why is this called `packed`? What's packed about it?

Reply:
Because all the CRDs are packed into one file, vs the originals where each resource is in a separate file.

I'm renaming the folder to `out` though, to better reflect it's an output of the compilation process.